src/App.js

```diff
@@ -309,6 +311,22 @@ export default function App(props) {
     }
   }
 
+  function saveAll() {
+    // Build zip file
+    const zip = new JSZip();
+    zip.file('FSH.fsh', inputFSHText);
+    inputFHIRText.forEach((def) => {
+      const name = `${def.id ?? 'Untitled'}.json`;
```

Comment:
This would be a rare scenario, but if there are two defs without `id`s, they are both saved as `Untitled`, so only one will remain, while others get overwritten.

Reply:
Ohh this is an excellent point and I'm not sure I'd even consider it rare (because I often add a bunch of blank definitions). I'll fix that!

Reply:
This should be resolved in 68e6668.

The edge case that I was testing when writing this can be recreated by:
- Using the FSH [here](https://fshschool.org/FSHOnline/#/share/39HmT5I)
- Adding (at least) three new, blank JSON files
- Adding `{ "id": "Untitled-1" }` to the last blank JSON file you created

This code should keep the provided ids for all JSON files with ids defined, should keep one of the blank files as "Untitled.json", and should append a number of any other "Untitled" JSON files that does not conflict with an existing file name.